tests/general/sealed-global-variables.json

```diff
@@ -106,6 +106,7 @@
         "cc.GFXFeature",
         "cc.GFXFilter",
         "cc.GFXFormat",
+        "cc.GFXFormatFeatureBit",
```

Comment:
I didn't see this have been exported anywhere, where have the global variable been assigned